session 03/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import ReactLogo from "./assets/react-logo.png";
+import UserIcon from "./assets/user.png";
+
+const headerUsingCreateElement = React.createElement(
+  "div",
+  { className: "title" },
+  [
+    React.createElement("h1", {}, "Header using React.CreateElement"),
+    React.createElement("h2", {}, "React BootCamp"),
+    React.createElement("h3", {}, "In 3 months"),
+  ]
+);
+
+var course = "React BootCamp";
```

Comment:
should use const here if value is not getting changed. Avoid using var, instead can use let where required.

Reply:
updated.